### PR TITLE
Minor words fix for README.zh-cn.md

### PR DESCRIPTION
**Pull request by zhenglaizhang**
Fix following minor issues:
1. One word missing: 我们将这样的系统之为响应式系统 => 我们将这样的系统**称**之为响应式系统
2. One word not consistent across the manifesto: 失败 => 故障
3. Update one adv as adj: 得到有效地处理 => 得到有效**的**处理

### CODE REVIEW

File: README.zh-cn.md [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 这些变化之所以会发生，是因为近几年的应用需求出现了戏剧性的变化。仅仅在几年之前，大型应用意味着数十台服务器，数秒的响应时间，数小时的离线维护时间以及若干 GB 的数据。而在今天，应用被部署在一切场合，从移动设备到基于云的集群，这些集群运行在数以千计的多核心处理器的之上。用户期望毫秒级的响应时间以及 100% 的正常运行时间。数据则以 PB 为单位来衡量。昨天的软件架构已经完全无法地满足今天的需求。
 
-我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统之为响应式系统。
+我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统称之为响应式系统。
```
Comment (hepin1989) [inline README.zh-cn.md]: good

File: README.zh-cn.md [hunk 2]
```diff
@@ -5,13 +5,13 @@
 
 这些变化之所以会发生，是因为近几年的应用需求出现了戏剧性的变化。仅仅在几年之前，大型应用意味着数十台服务器，数秒的响应时间，数小时的离线维护时间以及若干 GB 的数据。而在今天，应用被部署在一切场合，从移动设备到基于云的集群，这些集群运行在数以千计的多核心处理器的之上。用户期望毫秒级的响应时间以及 100% 的正常运行时间。数据则以 PB 为单位来衡量。昨天的软件架构已经完全无法地满足今天的需求。
 
-我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统之为响应式系统。
+我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统称之为响应式系统。
 
-以响应式系统方式构建的系统更加灵活，松耦合和[可扩展](/glossary#可扩展性)。这使得它们更容易被开发，而且经得起变化的考验。它们对于系统[失败](/glossary#失败)表现出显著的包容性，并且当失败真的发生时，它们能用优雅的方式去应对，而不是放任灾难的发生。响应式系统是高度灵敏的，能够给[用户](/glossary#用户)以有效的交互式的反馈。
+以响应式系统方式构建的系统更加灵活，松耦合和[可扩展](/glossary#可扩展性)。这使得它们更容易被开发，而且经得起变化的考验。它们对于系统[故障](/glossary#故障)表现出显著的包容性，并且当故障真的发生时，它们能用优雅的方式去应对，而不是放任灾难的发生。响应式系统是高度灵敏的，能够给[用户](/glossary#用户)以有效的交互式的反馈。
```
Comment (hepin1989) [inline README.zh-cn.md]: 以响应式系统`的`方式构建的系统更加灵活,how about add the `的`
`故障` is okay.

File: README.zh-cn.md [hunk 3]
```diff
@@ -5,13 +5,13 @@
...
-我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统之为响应式系统。
+我们相信，一种条理分明的系统架构方法是必要的，而且我们相信关于这种方法的所有必要方面已经逐一地被人们认识到：我们需要的系统是响应式的，具有可回复性的，可伸缩的，以及以消息驱动的。我们将这样的系统称之为响应式系统。
 
-以响应式系统方式构建的系统更加灵活，松耦合和[可扩展](/glossary#可扩展性)。这使得它们更容易被开发，而且经得起变化的考验。它们对于系统[失败](/glossary#失败)表现出显著的包容性，并且当失败真的发生时，它们能用优雅的方式去应对，而不是放任灾难的发生。响应式系统是高度灵敏的，能够给[用户](/glossary#用户)以有效的交互式的反馈。
+以响应式系统方式构建的系统更加灵活，松耦合和[可扩展](/glossary#可扩展性)。这使得它们更容易被开发，而且经得起变化的考验。它们对于系统[故障](/glossary#故障)表现出显著的包容性，并且当故障真的发生时，它们能用优雅的方式去应对，而不是放任灾难的发生。响应式系统是高度灵敏的，能够给[用户](/glossary#用户)以有效的交互式的反馈。
 
 **响应式系统是：**
 
-* <a name="灵敏的"></a>**灵敏的**：只要有可能，[系统](/glossary#系统)就会及时响应。灵敏性是可用性和效用的基石，但还不止于此，灵敏性还意味着问题能够被更快地侦测到并得到有效地处理。灵敏的系统着眼于提供迅速和一致的响应时间，建立可靠的服务上限，因而它们可以交付一致的服务质量。这种一致的行为反过来又能简化出错处理，建立最终用户对系统的信心，并且促使他们与系统做进一步交互。
+* <a name="灵敏的"></a>**灵敏的**：只要有可能，[系统](/glossary#系统)就会及时响应。灵敏性是可用性和效用的基石，但还不止于此，灵敏性还意味着问题能够被更快地侦测到并得到有效的处理。灵敏的系统着眼于提供迅速和一致的响应时间，建立可靠的服务上限，因而它们可以交付一致的服务质量。这种一致的行为反过来又能简化出错处理，建立最终用户对系统的信心，并且促使他们与系统做进一步交互。
```
Comment (hepin1989) [inline README.zh-cn.md]: typo fixed.

**Comment by hepin1989** [comment]
@zhenglaizhang I think you changes are fine.
@ktoso FYI.

**Comment by jboner** [comment]
@hepin1989 @zhenglaizhang Is this ready for merging now? Thanks a lot for the help.

**Comment by zhenglaizhang** [comment]
Sorry for the long delay response, I was busy with new job, learning new skills to write reactive system with scala :)

@hepin1989 I have just updated `以响应式系统方式` as `以响应式系统的方式`, plz help review if necessary.

@jboner I am fine with the merging, thanks for the coordination!

**Comment by ktoso** [comment]
Seems we're good to go here @jboner ? :)

**Pull request merged**